<launch>

  <!-- Sets the image source -->
  <remap from="image" to="/camera/image_raw" />

  <node pkg="ros_markers" type="detect" name="ros_markers">

    <!-- Sets the TF frame of the camera. -->
    <param name="camera_frame_id" type="str" value="my_camera" />

    <!-- Defines which and where are the markers in the environment. The sample
    file (markers_configuration_sample.yml) provides complete documentation.
    -->
    <param name="markers_configuration" type="str" value="$(find ros_markers)/config/markers_configuration_sample.yml" />

	<!-- set whether or not to ignore markers not included in the 
	configuration file (default false).    -->
    <param name="omit_other_tags" type="bool" value="true" />

    <!-- set the default size of markers, in millimeters, which are not 
    in the configuration file. -->
    <param name="default_marker_size" type="double" value="30.0" />

    <!-- set the level of filtering applied on the detected markers. 1.0 means
    no filtering, while 0.0 is the maximum level of filtering. 
    (-1 uses chilitags default value). -->
    <param name="gain" type="double" value="-1" />
    
    <!-- set the number of frames an object/marker should be not detected
    for before it is removed. (-1 uses chilitags default value). -->
    <param name="persistence" type="int" value="-1" />

  </node>

</launch>
